<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quadcoptor Debug Console</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="consoleTab">
       <attribute name="title">
        <string>Console</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="consoleHLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Settings</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="1">
              <widget class="QComboBox" name="portSelection"/>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="baudRateSelection">
               <property name="currentIndex">
                <number>16</number>
               </property>
               <item>
                <property name="text">
                 <string>50</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>75</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>110</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>134</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>150</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>300</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>19200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>38400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>57600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>115200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>230400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>460800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>500000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>576000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>921600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1000000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1152000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1500000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2000000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2500000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3000000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3500000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4000000</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="dataBitSelection">
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="paritySelection">
               <item>
                <property name="text">
                 <string>none</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>even</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>odd</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Port</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Baud Rate</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Data bits</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Parity</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QCheckBox" name="stopBitCheck">
               <property name="text">
                <string>2 Stop bit</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="2">
              <widget class="QPushButton" name="openButton">
               <property name="text">
                <string>Open</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0" colspan="2">
              <widget class="QPushButton" name="closeButton">
               <property name="text">
                <string>Close</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0" colspan="2">
              <widget class="QPushButton" name="clearButton">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="terminalLayout">
            <item row="0" column="0">
             <widget class="Terminal" name="terminal"/>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="lineLayout">
              <item>
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="Send">
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="plotTab">
       <attribute name="title">
        <string>Plot</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QWidget" name="widget" native="true">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCustomPlot" name="plotWidget1" native="true">
                <property name="minimumSize">
                 <size>
                  <width>700</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayout_5">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>kP</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSlider" name="kpSlider">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>kI</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="kiSlider">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>kD</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSlider" name="kdSlider">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="setPointLabel_6">
                  <property name="text">
                   <string>SetPoint</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDial" name="setPointDial_3"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QCustomPlot" name="plotWidget2" native="true">
                <property name="minimumSize">
                 <size>
                  <width>700</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayout_3">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>kP</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSlider" name="horizontalSlider_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>kI</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="horizontalSlider_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>kD</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSlider" name="horizontalSlider_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="setPointLabel_2">
                  <property name="text">
                   <string>SetPoint</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDial" name="setPointDial"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QCustomPlot" name="plotWidget3" native="true">
                <property name="minimumSize">
                 <size>
                  <width>700</width>
                  <height>0</height>
                 </size>
                </property>
                <zorder>formLayoutWidget_3</zorder>
               </widget>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayout_4">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>kP</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSlider" name="horizontalSlider_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>kI</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="horizontalSlider_8">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>kD</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSlider" name="horizontalSlider_9">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="setPointLabel_4">
                  <property name="text">
                   <string>SetPoint</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDial" name="setPointDial_2"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
          <zorder>widget_3</zorder>
          <zorder>widget_2</zorder>
          <zorder>widget_4</zorder>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Terminal</class>
   <extends>QTextEdit</extends>
   <header>terminal.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
